internals/cli/cli.go

```diff
@@ -53,62 +53,71 @@ type options struct {
 	Version func() `long:"version"`
 }
 
-type argDesc struct {
-	name string
-	desc string
+// PositionalArg contains information about the positional arguments accepted
+// by a command.
+type PositionalArg struct {
+	// Name of the positional argument as displayed in the command usage string.
+	Name string
+	// ShortHelp is a single-line help string that will be displayed in the
+	// command's manual alongside the argument name.
+	ShortHelp string
 }
 
 var optionsData options
 
 // ErrExtraArgs is returned  if extra arguments to a command are found
 var ErrExtraArgs = fmt.Errorf("too many arguments for command")
 
-// cmdInfo holds information needed to call parser.AddCommand(...).
-type cmdInfo struct {
-	name, shortHelp, longHelp string
-	builder                   func() flags.Commander
-	hidden                    bool
-	optDescs                  map[string]string
-	argDescs                  []argDesc
-	alias                     string
-	extra                     func(*flags.Command)
+// CmdInfo holds information needed by the CLI to execute commands and
+// populate entries in the help manual.
+type CmdInfo struct {
+	// Name of the command
+	Name string
+
+	// ShortHelp is a single-line help string that will be displayed
+	// in the full Pebble help manual (i.e. help --all)
+	ShortHelp string
+
+	// LongHelp contains exhaustive documentation about the command,
+	// that will be reflected in the specific help manual for the
+	// command, and in the Pebble man page.
+	LongHelp string
```

Comment:
Potentially `Summary` and `Description`? These are nice single-word descriptions of what each field represents, and in line with terminology we use elsewhere, for example in the Pebble layer config.

Reply:
Lovely! Thank you